src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -28,7 +28,7 @@ export default () => `
 				<div class="section deploy">
 					<h2 class="caption">${escape(localize('welcomePage.deploy', "Deploy"))}</h2>
 					<ul>
-						<li><a href="command:azdata.resource.deploy">${escape(localize('welcomePage.newDeployment', "New Deployment…"))}</a></li>
+						<li><a href="command:resourceDeployment.new">${escape(localize('welcomePage.newDeployment', "New Deployment…"))}</a></li>
```

Comment:
maybe add a note that if this is removed from the welcome page we should remote the action